pyproject.toml

```diff
@@ -0,0 +1,108 @@
+[tool.poetry]
+name = "mentorship-backend"
+version = "1.0.0"
+description = " Backend REST API for the Mentorship System."
+authors = ["Isabel Costa <unknown>"]
+maintainers = ["Isabel Costa <unknown>"]
+license = "GNU GPL v3"
+readme = "README.md"
+homepage = "https://github.com/anitab-org/mentorship-backend"
+repository = "https://github.com/anitab-org/mentorship-backend"
+documentation = "https://github.com/anitab-org/mentorship-backend/wiki"
+keywords = ["REST API",
+    "Anita Borg",
+    "Poetry",
+    "Flask",
+    "Women in Technology"]
+
+[tool.poetry.dependencies]
+python = "^3.6"
+alembic = "^1.4.2"
+aniso8601 = "^3.0.0"
+appdirs = "^1.4.3"
+APScheduler = "^3.5.1"
+asn1crypto = "^0.24.0"
+attrs = "^19.3.0"
+awsebcli = "^3.14.1"
+bcrypt = "^3.1.7"
+blessed = "^1.15.0"
+blinker = "^1.4"
+botocore = "^1.10.48"
+cached-property = "^1.4.3"
+cement = "^2.8.2"
+certifi = "^2018.4.16"
+cffi = "^1.11.5"
+chardet = "^3.0.4"
+click = "^6.7"
+colorama = "^0.3.9"
+coverage = "^4.5.1"
+cryptography = "^2.8"
+docker = "^3.4.0"
+docker-compose = "^1.21.2"
+docker-pycreds = "^0.3.0"
+dockerpty = "^0.4.1"
+docopt = "^0.6.2"
+docutils = "^0.14"
+entrypoints = "^0.3"
+Flask = "^1.0.2"
+Flask-JWT-Extended = "^3.10.0"
+Flask-Mail = "^0.9.1"
+Flask-Migrate = "^2.5.2"
+flask-restx = "^0.2.0"
+Flask-SQLAlchemy = "^2.3.2"
+Flask-Testing = "^0.7.1"
+future = "^0.16.0"
+gunicorn = "^20.0.4"
+idna = "^2.6"
+itsdangerous = "^0.24"
+Jinja2 = "^2.10"
+jmespath = "^0.9.3"
+jsonschema = "^2.6.0"
+Mako = "^1.1.3"
+MarkupSafe = "^1.1.1"
+mccabe = "^0.6.1"
+paramiko = "^2.7.1"
+pathspec = "^0.5.5"
+psycopg2-binary = "^2.8.5"
+pycodestyle = "^2.5.0"
+pycparser = "^2.18"
+pyflakes = "^2.1.1"
+PyJWT = "^1.4.2"
+PyMySQL = "^0.9.2"
+PyNaCl = "^1.3.0"
+python-dateutil = "^2.7.3"
+python-dotenv = "^0.8.2"
+python-editor = "^1.0.4"
+pytz = "^2018.4"
+PyYAML = "^3.13"
+regex = "^2020.2.20"
+requests = "^2.18.4"
+semantic-version = "^2.5.0"
+six = "^1.11.0"
+SQLAlchemy = "^1.2.7"
+tabulate = "^0.7.5"
+termcolor = "^1.1.0"
+texttable = "^0.9.1"
+toml = "^0.10.0"
+typed-ast = "^1.4.1"
+tzlocal = "^1.5.1"
+urllib3 = "^1.22"
+wcwidth = "^0.1.7"
+websocket-client = "^0.48.0"
+Werkzeug = "^0.15.3"
```

Comment:
Do we need all of the pacakages here? I am pretty sure some of them are actually sub deps of other packages, that have creep'd into here (more reason for doing this :slightly_smiling_face: )

Reply:
I'll go through them and remove the subdeps:)